test/token-paymaster/biconomy-token-paymaster-specs.ts

```diff
@@ -574,20 +574,15 @@ describe("Biconomy Token Paymaster", function () {
         "nonce"
       );
 
```

Comment:
New test cases missing to cover new logic.

Reply:
I checked it in other branch by making transfer again in else case of postOp. not sure if I can add a test in hardhat environment where there is another op which empties the tokens before main op gets mined, or user transfers token in midst of validation and execution.